README.md

```diff
@@ -1,21 +1,18 @@
 # post-covid-respiratory
 
-[View on OpenSAFELY](https://jobs.opensafely.org/repo/https%253A%252F%252Fgithub.com%252Fopensafely%252Fpost-covid-respiratory/)
+## Repository navigation
 
-Details of the purpose and any published outputs from this project can be found at the link above.
+-   If you are interested in how we defined our code lists, look in the [`codelists`](./codelists) folder.
 
-The contents of this repository MUST NOT be considered an accurate or valid representation of the study or its purpose. 
-This repository may reflect an incomplete or incorrect analysis with no further ongoing work.
-The content has ONLY been made public to support the OpenSAFELY [open science and transparency principles](https://www.opensafely.org/about/#contributing-to-best-practice-around-open-science) and to support the sharing of re-usable code for other subsequent users.
-No clinical, policy or safety conclusions must be drawn from the contents of this repository.
+-   Analyses scripts are in the [`analysis`](./analysis) directory:
 
-# About the OpenSAFELY framework
+    -   If you are interested in how we defined our variables, we use the variable script [variable_helper_fuctions](analysis/variable_helper_functions.py) to define functions that generate variables. We then apply these functions in [variables_cohorts](analysis/variables_cohorts.py) to create a dictionary of variables for cohort definitions, and in [variables_dates](analysis/variables_dates.py) to create a dictionary of variables for calculating study start dates and end dates.
+    -   If you are interested in how we defined study dates (e.g., index and end dates), these vary by cohort and are described in the protocol. We use the script [dataset_definition_dates](analysis/dataset_definition_dates.py) to generate a dataset with all required dates for each cohort. This script imported all variables generated from [variables_dates](analysis/variables_dates.py).
+    -   If you are interested in how we defined cohorts, we use dataset definition script [dataset_definition_cohorts](analysis/dataset_definition_cohorts.py) to define the function that generate cohort. This script imported all variables generated from [variables_cohorts](analysis/variables_cohorts.py). We extracted the respective index dates and end dates for each cohort and applied this function to generate three specific cohorts—pre-vaccination, vaccinated, and unvaccinated—found in [dataset_definition_prevax](analysis/dataset_definition_prevax.py), [dataset_definition_vax](analysis/dataset_definition_vax.py), and [dataset_definition_unvax](analysis/dataset_definition_unvax.py). Extracted data is then combined to create our final cohorts, in the [preprocess data script](analysis/preprocess_data.R).
```

Comment:
Minor typo corrections below:

- If you are interested in how we defined our cohorts, we use the dataset definition script [dataset_definition_cohorts](analysis/dataset_definition_cohorts.py) to define a function that generates cohorts. This script imports all variables generated from [variables_cohorts](analysis/variables_cohorts.py) using the cohort index and end dates. This approach is used to generate three cohorts: pre-vaccination, vaccinated, and unvaccinated—found in [dataset_definition_prevax](analysis/dataset_definition_prevax.py), [dataset_definition_vax](analysis/dataset_definition_vax.py), and [dataset_definition_unvax](analysis/dataset_definition_unvax.py) respectively.

Reply:
I am not sure what combined is referring to in this sentence: 'Extracted data is then combined to create our final cohorts, in the [preprocess data script](analysis/preprocess_data.R).'?

Reply:
Directly copied from the mental health repo . Indeed nothing to combine for the current ehrQL repo. 
I will just say:
The extracted data is initially processed in the preprocess data script [preprocess data script](analysis/preprocess_data.R) to clean consultation rates, create a categorical variable for severe COVID-19, generate a flag variable for pre-existing respiratory conditions, and restrict columns, resulting in the final cohort.

Reply:
Some futher edits below:

> -   If you are interested in how we defined our cohorts, we use the dataset definition script [dataset_definition_cohorts](analysis/dataset_definition_cohorts.py) to define a function that generates cohorts. This script imports all variables generated from [variables_cohorts](analysis/variables_cohorts.py) using the patient's index date, the cohort start date and the cohort end date. This approach is used to generate three cohorts: pre-vaccination, vaccinated, and unvaccinated—found in [dataset_definition_prevax](analysis/dataset_definition_prevax.py), [dataset_definition_vax](analysis/dataset_definition_vax.py), and [dataset_definition_unvax](analysis/dataset_definition_unvax.py), respectively. For each cohort, the extracted data is initially processed in the [preprocess data script](analysis/preprocess_data.R), which cleans consultation rate, creates a categorical variable for severe COVID-19, generates a flag variable for pre-existing respiratory conditions, and restricts the data to relevant variables.